templates/tools/dockerfile/test/cxx_clang_12_x64/Dockerfile.template

```diff
@@ -0,0 +1,28 @@
+%YAML 1.2
+--- |
+  # Copyright 2021 gRPC authors.
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  
+  FROM silkeh/clang:12
```

Comment:
who's silkeh and why did you choose images from silkeh/*. I'd prefer if we used something more official than a random person from github who happens to have built clang images.

Reply:
fair enough, perhaps you could add a comment?